service/patch.go

```diff
@@ -25,9 +25,9 @@ func (uis *UIServer) patchPage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	currentUser := MustHaveUser(r)
-
+	spruceLink := fmt.Sprintf("%s/patch/%s/configure", uis.Settings.Ui.UIv2Url, projCtx.Patch.Id.Hex())
```

Comment:
For this and the other places that set it, I would only set if the UIv2 URL is not empty